### DISCUSSION TITLE
Mongo indexes

### DISCUSSION THREAD

**Discussion by MattSzm** [monix/monix-connect#904] [Q&A] [answered]
Hello, I'm using monix connector and I'm wondering how to create any custom index. Is there any defined way of doing this or at least some hack? It seems to be a crucial feature of any db.

**Comment by paualarco** [comment, marked as answer]
Hey @MattSzm, thanks for raising the question, it's definitely something we would like to add to the existing api.
Have created an issue: https://github.com/monix/monix-connect/issues/907.
Are you maybe interested in contributing? :)

**Reply by MattSzm** [reply]
I've created a PR: https://github.com/monix/monix-connect/pull/909
